src/main/java/cart/dto/request/CartItemInfoRequest.java

```diff
@@ -0,0 +1,17 @@
+package cart.dto.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class CartItemInfoRequest {
+
+    private final Long cartItemId;
+    private final Integer quantity;
+    private final ProductInfoRequest product;
+
+    private CartItemInfoRequest() {
+        this(null, null, null);
```

Comment:
```suggestion
        this(null, null, null);
```
일반적이지 않은 코드같은데요
우선 롬복을 해제하고 생각해보죠!

Reply:
빈 생성자가 parsing할때 꼭 필요 하던가요??
보통은 final을 빼고 빈생성자를 만들거나
빈생성자를 없애는 방향을 택하는것 같아요

Reply:
인텔리제이에서 build tool로 intelliJ를 선택하면 발생하는 에러 때문에 레벨2 두 번째 미션부터 관습적으로 작성해왔습니다.

필요없는 것은 사실이지만 intelliJ를 선택해 사용하는 개발자들이 있기에 배려하고자는 차원에서 작성했습니다.
사실 저번 미션에서도 같은 부분으로 피드백을 받았었는데요. 불필요한 배려일까요..?